website/pages/api-docs/volumes.mdx

```diff
@@ -0,0 +1,235 @@
+---
+layout: api
+page_title: Volumes - HTTP API
+sidebar_title: Volumes
+description: The `/volume` endpoints are used to query for and interact with volumes.
+---
+
+# Volumes HTTP API
+
+The `/volume` endpoints are used to query for and interact with volumes.
+
+## List Volumes
+
+This endpoint lists all volumes.
+
+| Method | Path          | Produces           |
+| ------ | ------------- | ------------------ |
+| `GET`  | `/v1/volumes` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs#blocking-queries) and
+[required ACLs](/api-docs#acls).
+
+| Blocking Queries | ACL Required                |
+| ---------------- | --------------------------- |
+| `YES`            | `namespace:csi-list-volume` |
+
+### Parameters
+
+- `type` `(string: "")`- Specifies a string to filter volumes based on
+  the volume type. Currently supports only `csi`. This is specified as
+  a query string parameter.
+
+- `node_id` `(string: "")` - Specifies a string to filter volumes
+  based on an Node ID prefix. Because the value is decoded to bytes,
+  the prefix must have an even number of hexadecimal characters
+  (0-9a-f). This is specified as a query string parameter.
+
+- `plugin_id` `(string: "")` - Specifies a string to filter volumes
+  based on a plugin ID prefix. Because the value is decoded to bytes,
+  the prefix must have an even number of hexadecimal characters
+  (0-9a-f). This is specified as a query string parameter.
+
+### Sample Request
+
+```shell
+$ curl \
+    https://localhost:4646/v1/volumes
```

Comment:
This returns an empty list, the `type` must be provided to get results

Reply:
I've removed the example for both volumes and plugins where the query param has been omitted.